tests/e2e/test-data/rdm-full.json

```diff
@@ -0,0 +1,219 @@
+[
```

Comment:
What is that file?
Please add a description on top

Reply:
no, it's json file

Reply:
which is only used to uploading this file in app